src/main/resources/com/devonfw/ide/sonarqube/common/rules/devon4j/devon4j.xml

```diff
@@ -9668,7 +9661,7 @@
         <repositoryKey>java</repositoryKey>
         <key>S2699</key>
         <type>CODE_SMELL</type>
-        <priority>BLOCKER</priority>
+        <priority>INFO</priority>
```

Comment:
this change could be merged already. Maybe it would be better to create small individual PRs...

Reply:
Sure, I will create another PR for the devon4j.xml changes.